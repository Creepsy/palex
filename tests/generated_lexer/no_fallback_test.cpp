#include <iostream>
#include <string_view>

#include <NoFallbackTestLexer.h>

int main() {
    const std::string_view input = "inte";
    palex::NoFallbackTestLexer lexer(input);
    do {
        lexer.next_unignored_token();
        std::cout << lexer.current_token() << std::endl;
    } while (!lexer.end());
    return 0;
}