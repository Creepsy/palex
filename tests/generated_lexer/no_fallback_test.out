[Ln 1, Col 1] - [Ln 1, Col 5] UNDEFINED: inte
